From e8796a001ea4ce034d8209f726686e10945099af Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Wed, 20 Mar 2024 14:18:30 +0000
Subject: [PATCH 0984/1085] drivers: mmc: trigger activity LED when CQE is
 active

Add a LED_FULL trigger equivalent to mmc_start_request() in
mmc_cqe_start_req(), otherwise it stays off forever.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 drivers/mmc/core/core.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/mmc/core/core.c
+++ b/drivers/mmc/core/core.c
@@ -455,6 +455,7 @@ int mmc_cqe_start_req(struct mmc_host *h
 		goto out_err;
 
 	trace_mmc_request_start(host, mrq);
+	led_trigger_event(host->led, LED_FULL);
 
 	return 0;
 
